mlir/tools/mlir-tblgen/OpFormatGen.cpp

```diff
@@ -1654,12 +1654,12 @@ void OperationFormat::genElementParser(FormatElement *element, MethodBody &body,
           dir->shouldBeQualified() ? qualifiedTypeParserCode : typeParserCode;
       TypeSwitch<FormatElement *>(dir->getArg())
           .Case<OperandVariable, ResultVariable>([&](auto operand) {
-            body << formatv(parserCode,
+            body << formatv(false, parserCode,
```

Comment:
OOC is validation false here as it would currently fail to validate?

Reply:
Yeah, because parserCode is either with 1 or 2 placeholders:

```
      const char *parserCode =
          dir->shouldBeQualified() ? qualifiedTypeParserCode : typeParserCode;
```

We could add if's in the code around calls to formatv() as well, but I though the code duplication just to get validation was not warranted.